### PR TITLE
feat: avg aggregation sql runner

### PR DESCRIPTION
**Pull request by rephus**
Closes:   #11139

### Description:

Screenshot from 2024-08-30 14-54-59

### Reviewer actions

- [ ] I have manually tested the changes in the preview environment
- [ ] I have reviewed the code
- [ ] I understand that "request changes" will block this PR from merging

### CODE REVIEW

File: packages/frontend/src/components/DataViz/config/DataVizAggregationConfig.tsx [hunk 1]
```diff
@@ -30,6 +30,7 @@ const AggregationIcon: FC<{ aggregation: string | undefined }> = ({
             icon = IconSum;
             break;
         case MetricType.AVERAGE:
+        case VizAggregationOptions.AVERAGE:
             icon = IconTrendingUp;
```
Comment (rephus) [inline packages/frontend/src/components/DataViz/config/DataVizAggregationConfig.tsx:34]: I don't think this is a great icon for `average` , but I can't find a find a better one to replace it

Comment (joaoviana) [reply packages/frontend/src/components/DataViz/config/DataVizAggregationConfig.tsx:34]: [IconMathAvg](https://tabler.io/icons/icon/math-avg)

This apparently 😅

Comment (rephus) [reply packages/frontend/src/components/DataViz/config/DataVizAggregationConfig.tsx:34]: You think ? that doesn't look right to me..

Comment (joaoviana) [reply packages/frontend/src/components/DataViz/config/DataVizAggregationConfig.tsx:34]: I mean the name of the icon has avg, but I don't see it being widely used as **the** symbol for average. what you have works for me, checking the code!

**Review by joaoviana — APPROVED** [review]
Just needs a type fix, but functionality works great!

File: packages/backend/src/services/SemanticLayerService/Pivoting.ts [hunk 2]
```diff
@@ -44,9 +49,15 @@ export const groupResults = (
 ): SemanticLayerResultRow[] => {
     const df = pl.DataFrame(results);
 
-    const aliasedAggs: pl.Expr[] = values.map((v) =>
-        pl.col(v.name)[v.aggFunction]().alias(`${v.aggFunction}(${v.name})`),
-    );
+    const aliasedAggs: pl.Expr[] = values.map((v) => {
```
Comment (owlas) [inline packages/backend/src/services/SemanticLayerService/Pivoting.ts:52]: @almeidabbm and @IrakliJani why is this still here? I think it's confusing, we don't need this aggregation and developers shouldn't need to worry about it.

**Pull request merged**